webapp/channels/src/components/admin_console/system_users/system_users_list/index.tsx

```diff
@@ -288,10 +370,13 @@ function SystemUsersList(props: Props) {
             tableId: 'systemUsersTable',
             tableCaption: formatMessage({id: 'admin.system_users.list.caption', defaultMessage: 'System Users'}),
             loadingState,
+            disablePrevPage: !props.tablePropertyCursorUserId || props.tablePropertyPageIndex <= 0 || (props.tablePropertyCursorDirection === 'prev' && userReports.length < paginationTableState.pageSize),
+            disableNextPage: props.tablePropertyCursorDirection === 'next' && userReports.length < paginationTableState.pageSize,
             onRowClick: handleRowClick,
             onPreviousPageClick: handlePreviousPageClick,
             onNextPageClick: handleNextPageClick,
-            hasAdditionalPaginationAtTop: false,
+            paginationInfo: getPaginationInfo(paginationTableState.pageIndex, paginationTableState.pageSize, userReports.length, userCount ?? 0),
+            hasAdditionalPaginationAtTop: true,
             totalRowInfo: '',
```

Comment:
can you pass the total here as well

Reply:
<img width="1711" alt="Screenshot 2024-01-11 at 21 37 13" src="https://github.com/mattermost/mattermost/assets/17708702/a5d9ba23-3f97-455c-8258-3da33ef3f2a2">

@abhijit-singh this is how it is right now

Reply:
@M-ZubairAhmed where do you want me to pass it?